### ISSUE TITLE
Zoom is not inlay specific

### ISSUE THREAD

**Issue by pistolwhip-dev** [Styr1x/Browsingway#33]
Setting the Zoom parameter in individual overlays does not seem to stay applied to inlays on a per definition basis.

If I create 3 inlays A, B, and C:

A Zoom 100%
B Zoom 150% - While setting B, A will also zoom to 150% in real time
C Zoom 80%  - Both A and B will also adjust in real time

And on relaunch of the game, all three seem to land on the last defined value, so in this example C at 80%

This was tested and confirmed in a few different plugin sets, including in isolation (BW only installed).

**Comment by Styr1x** [comment]
Hey,
I'm unable to reproduce this;
Could you send me your %appdata%\XIVLauncher\pluginConfigs\Browsingway.json (if you have sensitive Information in your URLs remove them from the file beforehand).

**Comment by pistolwhip-dev** [comment]
Sure, github wouldn't let me attach a json so excuse the long post, but bog standard things I think:

```
{
  "$type": "Browsingway.Configuration, Browsingway",
  "FrameTransportMode": 1,
  "Inlays": [
    {
      "$type": "Browsingway.InlayConfiguration, Browsingway",
      "ClickThrough": false,
      "Framerate": 60,
      "Guid": "d9d7a365-3d1a-42dc-b75b-7a2f4427cb38",
      "Hidden": false,
      "Locked": false,
      "Name": "Cactbot Config",
      "Opacity": 100.0,
      "TypeThrough": false,
      "Url": "http://proxy.iinact.com/overlay/cactbot/ui/config/config.html?OVERLAY_WS=ws://127.0.0.1:10501/ws",
      "Zoom": 100.0,
      "Disabled": true,
      "Muted": false,
      "ActOptimizations": false
    },
    {
      "$type": "Browsingway.InlayConfiguration, Browsingway",
      "ClickThrough": true,
      "Framerate": 60,
      "Guid": "b7145174-8000-4c59-9db6-8c1f4a82040f",
      "Hidden": false,
      "Locked": true,
      "Name": "Expeds",
      "Opacity": 99.0,
      "TypeThrough": false,
      "Url": "http://proxy.iinact.com/overlay/cactbot/ui/eureka/eureka.html?OVERLAY_WS=ws://127.0.0.1:10501/ws",
      "Zoom": 100.0,
      "Disabled": false,
      "Muted": false,
      "ActOptimizations": false
    },
    {
      "$type": "Browsingway.InlayConfiguration, Browsingway",
      "ClickThrough": false,
      "Framerate": 30,
      "Guid": "8caee225-2901-4a86-bb8a-81a49875dbae",
      "Hidden": false,
      "Locked": true,
      "Name": "mopi",
      "Opacity": 100.0,
      "TypeThrough": true,
      "Url": "http://proxy.iinact.com/overlay/mopimopi/?HOST_PORT=ws://127.0.0.1:10501/",
      "Zoom": 73.0,
      "Disabled": false,
      "Muted": false,
      "ActOptimizations": false
    },
    {
      "$type": "Browsingway.InlayConfiguration, Browsingway",
      "ClickThrough": true,
      "Framerate": 30,
      "Guid": "4fa1ed83-127f-43d4-ab6a-23ed5edaeb18",
      "Hidden": false,
      "Locked": false,
      "Name": "raidboss",
      "Opacity": 100.0,
      "TypeThrough": false,
      "Url": "http://proxy.iinact.com/overlay/cactbot/ui/raidboss/raidboss.html?OVERLAY_WS=ws://127.0.0.1:10501/ws",
      "Zoom": 186.0,
      "Disabled": false,
      "Muted": false,
      "ActOptimizations": false
    }
  ],
  "Version": 0
}
```

**Comment by Styr1x** [comment]
Thanks;
I've tried your config and even with it I can't reproduce it on my side. I'm at a loss here to why this is happening, two of your overlays inside the config have different zoom levels set - when BW sends zoom to its renderer it also saves them, so I'd except them to be the same. Not sure how to solve this at this point.

**Comment by pistolwhip-dev** [comment]
Very odd indeed. I suppose I'll try completely clean-installing xivlauncher and dal.  Maybe something from my existing config is somehow interfering in a way I can't understand.  Perhaps 'disabled' isn't sufficient to completely unload plugins and one is interfering?  Either way, I'll give it a shot

**Comment by pistolwhip-dev** [comment]
Following up, I tried a completely vanilla install of xl and just bw.  Same behavior.  Interestingly, the 'chosen' zoom seems to have changed from the last defined inlay, to the first, which is again, quite odd.  I didn't see anything in dal logs either, even on verbose.

My system is pretty standard...I'm not running any external shader stuff, non-xl injections (alexander etc), or anything else really.  I'm using IINACT as a source for overlay data (or, trying to)

**Comment by pistolwhip-dev** [comment]
Still can't figure this out.  Shot a quick vid demonstrating the behavior.  This was done with a completely fresh install of Dalamud/Xivlauncher.  

https://user-images.githubusercontent.com/55765377/221424607-2a759805-eb15-4b47-84e2-2bde079c9a6f.mp4

**Comment by Styr1x** [comment]
Fixed with v1.5.2.0
